packages/workbox-webpack-plugin/src/lib/get-webpack-manifest-entries.js

```diff
@@ -0,0 +1,76 @@
+const url = require('url');
+
+const entry = (revision, filePath, publicPath) => ({
+  revision,
+  url: url.resolve(publicPath, filePath),
```

Comment:
I remember this coming up in a review of the earlier revision of the Webpack plugin: `publicPath` [is defined](https://webpack.js.org/configuration/output/#output-publicpath) as something that's prepended to every URL, not joined with it, so this should probably be `publicPath + filePath`;

Reply:
Oops, me too. Will change that